scheduler/src/cook/mesos/api.clj

```diff
@@ -2711,6 +2711,11 @@
           (update :body body-matchers)
           (update :response resp-matchers)))))
 
+(defn- logging-exception-handler [base-handler]
+  (fn logging-exception-handler [ex data req]
+    (log/error ex "Error when processing request" (dissoc req ::c-mw/options))
```

Comment:
We should be more restrictive about what we print in the request. Waiter has the following:

```
(-> (dissoc request :body :ctrl :request-time :server-name :server-port :servlet-request :ssl-client-cert :support-info)
      (update-in [:headers] headers/truncate-header-values))
```

Reply:
Done.